sdk/dotnet/Cloud/AwsGovcloudLinkAccount.cs

```diff
@@ -24,7 +24,6 @@ namespace Pulumi.NewRelic.Cloud
     /// 
     /// ## Example Usage
     /// 
-    /// &lt;!--Start PulumiCodeChooser --&gt;
```

Comment:
Are these dropped intentionally?

Reply:
Yes. That was https://github.com/pulumi/pulumi-terraform-bridge/pull/1887.